src/main/scala/Day4/LumberJack.scala

```diff
@@ -0,0 +1,50 @@
+package Day4
+
+object LumberJack extends App {
+  var gridSize: Int = 3
+  var numberOfLogs = 7
+  var logPile: List[Int] = List(1, 1, 1, 2, 1, 3, 1, 4, 1)
+  var count = 0
+
+  def lumberjack(): Unit = {
```

Comment:
all variables should be `val` you're not reassigning the value

additionally you're not using them afterwards so I'm assuming this is still work in progress?